networks.json

```diff
@@ -1,4 +1,42 @@
 {
+  "BatchExchange": {
```

Comment:
All these network changes were probably unintended since we have intentionally removed them quite recently... I would expect this file to remain unchanged.